creator-node/src/services/stateMachineManager/stateMonitoring/StateMonitoringQueue.js

```diff
@@ -173,55 +178,47 @@ class StateMonitoringQueue {
     })
   }
 
-  /**
-   * Registers the logic that gets executed to process each new job from the queue.
-   * @param {Object} queue the StateMonitoringQueue to consume jobs from
-   */
-  registerQueueJobProcessor(queue) {
-    // Initialize queue job processor (aka consumer)
-    queue.process(1 /** concurrency */, async (job) => {
-      const {
-        id: jobId,
-        data: {
-          lastProcessedUserId,
-          discoveryNodeEndpoint,
-          moduloBase,
-          currentModuloSlice
-        }
-      } = job
-
-      try {
-        this.log(`New job details: jobId=${jobId}, job=${JSON.stringify(job)}`)
-      } catch (e) {
-        this.logError(`Failed to log details for jobId=${jobId}: ${e}`)
+  async processJob(job) {
+    const {
+      id: jobId,
+      data: {
+        lastProcessedUserId,
+        discoveryNodeEndpoint,
+        moduloBase,
+        currentModuloSlice
       }
+    } = job
 
-      // Default results of this job will be passed to the next job, so default to failure
-      let result = {
+    try {
+      this.log(`New job details: jobId=${jobId}, job=${JSON.stringify(job)}`)
+    } catch (e) {
+      this.logError(`Failed to log details for jobId=${jobId}: ${e}`)
+    }
+
+    // Default results of this job will be passed to the next job, so default to failure
+    let result = {
+      lastProcessedUserId,
+      jobFailed: true,
+      moduloBase,
+      currentModuloSlice
+    }
+    try {
+      // TODO: Wire up metrics
+      // await redis.set('stateMachineQueueLatestJobStart', Date.now())
```

Comment:
this is copied from the snapback redis and would've involved a little more work to wire up to the health check so I left it as a TODO in the original refactor PR because it was already way too large of a PR. but I can go ahead and wire that up in the next PR